torch/_inductor/codecache.py

```diff
@@ -584,6 +587,11 @@ def cpp_flags():
 
 def optimization_flags():
     base_flags = "-O3 -ffast-math -fno-finite-math-only"
+    if config.is_fbcode():
```

Comment:
You need to coordinate with Brian on this.

Reply:
Yes, I will check with Brian. Thanks.